storybook/stubs/AppLayouts/Wallet/stores/RootStore.qml

```diff
@@ -35,4 +38,18 @@ QtObject {
     function getOpenSeaCollectibleUrl(networkShortName, contractAddress, tokenId) {
         return "https://somedummyurl.com"
     }
+
+    function getDappDetails(chainId, contractAddress) {
```

Comment:
As @caybro mentioned stub should be empty file. But here the problem is with singleton store. It should be fixed but ofc not as a part of this pr. We can live with this for now.